services/device-registry-file/src/main/java/org/eclipse/hono/deviceregistry/file/FileBasedServiceConfig.java

```diff
@@ -226,7 +179,7 @@ public FileBasedDeviceBackend deviceBackend() {
         autoProvisioner.setVertx(vertx);
         autoProvisioner.setTracer(tracer);
         autoProvisioner.setTenantInformationService(tenantInformationService);
-        autoProvisioner.setEventSender(eventSender());
+        autoProvisioner.setMessagingClients(messagingClients(SendMessageSampler.Factory.noop(), tracer, vertx, new ProtocolAdapterProperties()));
```

Comment:
passing in a *ProtocolAdapterProperties* instance from within a non-adapter component looks very, very strange